### ISSUE TITLE
Accessibility problems that make Telegram unusable

### ISSUE THREAD

**Issue by ghost** [TelegramMessenger/Telegram-iOS#163]
The problems I've listed make Telegram very useless.
We cannot use the application efficiently.

1. On the screens with more than 4-5 chat lists, the next items cannot be accessed with VoiceOver.

2. Telegram In all chat types supported by IOS, very few messages can be navigated.
Messages on the display remain constant.
For example, we cannot access past messages.

3. VoiceOver responds very late (2-3 seconds) to all interactive items except buttons such as "edit".
This may be due to the interface kit used by Telegram.

4. VoiceOver repeatedly says "access available" on screens with more than one item type.
Deactivating this feature will provide a healthier use.

5. There is an unlabeled button next to the "Send media" button. Button is not available with VoiceOver.

6. heads can sometimes be accessed, sometimes not, in settings and many screens.

**Comment by destroyersboy** [comment]
Hey, I guess nobody sees this. But these are important.

Accessibility is important to meet the free world.

Should I write these issues to the telegram team elsewhere?

I am very sad.  I don't have any c and  Swift knowledge, I'm a poor without a Mac.
